src/matrix_cookbook/9_special_matrices.lean

```diff
@@ -117,46 +395,46 @@ end
 
 /-! ### Nilpotent -/
 
-lemma eq_428 : sorry := sorry
+-- lemma eq_428 : sorry := sorry
```

Comment:
I realize I've not been particularly consistent, but I'd rather leave these uncommented as that detects errors where I wrote the same number twice.

If they're annoying when editing, you can just write `#exit` on the line above them to stop Lean processing them.